config/webpack/webpack.config.js

```diff
@@ -133,6 +134,10 @@ const webpackConfig = (options, env, argv) => {
           'terra-navigation-prompt',
         ],
       }),
+      new webpack.DefinePlugin({
+        PACKAGE_VERSION: JSON.stringify(process.env.npm_package_version),
```

Comment:
is process.env.npm_package_version available when running in yarn?

Reply:
will this require use to release and/or bump the client package-json in out servers for this version to change?

Reply:
1. I think so? I see package.json vars referenced within their documentation.

2. Yes, that version would need to be updated. Given that, I don't think the version we're providing here would be all useful with how we're deploying applications today. I kept it in because it _could_ be helpful in some ways, and just exposing it isn't hurting anything, but I also wouldn't argue over just removing it in the meantime. The build date is much more necessary.

Reply:
Went ahead and removed version for the time being